### ISSUE TITLE
libva update broke steam

### ISSUE THREAD

**Issue by nonchip** [void-linux/void-packages#4178]
since the libva update to version 2, `/usr/lib{32,64}/libva*.so.1` is (obviously) missing, but depended upon by steam, which on startup shows the following warning:
```
You are missing the following 32-bit libraries, and Steam may not run:
libva.so.1
libva.so.1
libva-x11.so.1
Press enter to continue: 
```
and then crashes with a popup of `Fatal Error: Failed to load steamui.so`.

so yeah i think we kinda need a bunch of `libva1*` packages.

**Comment by nonchip** [comment]
yeah except for one lutris doesn't support flatpacked steam, and also some people don't like running osx app style abominations on their systems.... also why'd that need a new ffmpeg? i didn't see you updating ffmpeg in that commit?

**Comment by maxice8** [comment]
> yeah except for one lutris doesn't support flatpacked steam

Seems like they are working on it.

> also why'd that need a new ffmpeg?

https://github.com/ValveSoftware/steam-for-linux/issues/5339

> i didn't see you updating ffmpeg in that commit?

Because it is an anti-pattern to include unrelated changes in a commit

**Comment by nonchip** [comment]
ok that seems to be a problem then yep.

**Comment by nonchip** [comment]
> Seems like they are working on it.

until they fixed it, in case anyone needs a workaround:

```sh
#!/bin/bash
#file: /usr/local/bin/flatsteam
exec flatpak run --filesystem=$HOME/.local/share/Steam com.valvesoftware.Steam "$@"
```
![steam](https://user-images.githubusercontent.com/601018/47721539-a21b6980-dc50-11e8-942d-6b678bf90c89.png)

**Comment by klaus1488** [comment]
Same problem here.

**Comment by Johnnynator** [comment]
@klaus1488 It should also work by not disabling the steam runtime.

**Comment by nonchip** [comment]
@Johnnynator well i never disabled it when i had that issue, so if it works now it's because something got fixed. probably just them adding their old ffmpeg+libva to the runtime
